.github/workflows/build.yml

```diff
@@ -320,6 +320,8 @@ jobs:
             fi
           else
             TEST_SUITE_TAGS="all"
+            # On scheduled runs, we only run one test at a time
+            TEST_FLAGS="-p 1"
```

Comment:
Leaving this here but I'm not sure if we want to keep it.

Reply:
Let's drop it. This seems like it would slow down all our tests significantly and if we don't have a strong use-case for it it doesn't seem worth it.